packages/currency/package.json

```diff
@@ -44,6 +44,8 @@
     "@metamask/contract-metadata": "1.28.0",
     "@requestnetwork/types": "0.35.0",
     "@requestnetwork/utils": "0.35.0",
+    "@types/multicoin-address-validator": "0.5.0",
```

Comment:
maybe a detail but this should be a devDependency

Reply:
oops! done